src/cli/rustup_mode.rs

```diff
@@ -111,6 +112,12 @@ pub fn cli() -> App<'static, 'static> {
         .setting(AppSettings::VersionlessSubcommands)
         .setting(AppSettings::DeriveDisplayOrder)
         .setting(AppSettings::SubcommandRequiredElseHelp)
+        .arg(
+            Arg::with_name("home")
+                .help("Print $RUSTUP_HOME")
+                .short("h")
+                .long("home"),
+        )
```

Comment:
Why is this arg here at all?  this would be `rustup --home` which is not something I really want, nor are you really processing it anyway.

Reply:
Sorry, that was an artifact of a previous attempt. I forgot to remove it.